src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import circleShadow from './CIRCLE_BUTTON_SHADOW.svg';
+import linkedinIcon from './LINKEDINLOGO.svg';
+import githubIcon from './GITHUBLOGO.svg';
+
+class Footer extends Component {
+  render() {
+    return (
+      <footer className="wrapper pt-5 pb-5 flex justify-between items-center">
+        <div className="flex">
+          <h4>Open Library Project</h4>
+        </div>
+        <div className="links flex">
+          <ShowOffLink
```

Comment:
I am curious why this name for the component?

Reply:
Probably not the right phrasal verb :D Changed it!